src/main/java/uk/gov/hmcts/sscs/email/RoboticsEmail.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.sscs.email;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class RoboticsEmail extends Email {
+
+    public RoboticsEmail(@Value("${appeal.email.from}") String from,
+                         @Value("${appeal.email.to}") String to,
+                         @Value("${appeal.email.subject}") String subject,
+                         @Value("${appeal.email.message}") String message) {
+        this.from = from;
```

Comment:
The application properties should 
robotics.email.to
robotics.email.subject
robotics.email.message
robotics.email.enabled